src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -292,10 +424,18 @@ public Server build() {
             blockingTaskExecutor = defaultBlockingTaskExecutor();
         }
 
+        final List<ServerPort> ports =
```

Comment:
Added the behavior about when port() was never called to the Javadoc of port() methods.